app/Http/Requests/PositionFormRequest.php

```diff
@@ -21,10 +23,40 @@ public function authorize()
      *
      * @return array
      */
-    public function rules()
+    public static function rulesStore($request)
     {
-        return [
-            'name' => 'required|min:2|max:255|unique:positions,name,' . $this->id,
+        $messages = [
+            'name.required' => __('Position') . __('Required'),
+            'name.string' => __('Position') . __('String'),
+            'name.unique' => __('Position') . __('Unique'),
         ];
+        
+        return  Validator::make($request->all(), [
+                'name' => 'required|string|unique:positions',
+            ], $messages
+        );
+    }
+
+    public static function rulesUpdate($request, $id) 
```

Comment:
**Reporter**: CHECKSTYLE</br>**Rule**: Squiz.WhiteSpace.SuperfluousWhitespace.EndLine</br>**Severity**: error</br>**File**: app/Http/Requests/PositionFormRequest.php</br>**Line**: 40</br>Whitespace found at end of line